director/client_request.go

```diff
@@ -106,11 +106,19 @@ func (r ClientRequest) RawGet(path string, out io.Writer, f func(*http.Request))
 		return nil, nil, bosherr.WrapErrorf(err, "Performing request GET '%s'", url)
 	}
 
-	return r.readResponse(resp, out)
+	return r.readResponse(resp, out, false)
 }
 
 // RawPost follows redirects via GET unlike generic HTTP clients
 func (r ClientRequest) RawPost(path string, payload []byte, f func(*http.Request)) ([]byte, *http.Response, error) {
+	return r.rawPostInternal(path, payload, f, false)
+}
+
+func (r ClientRequest) RawPostIgnoreNotFound(path string, payload []byte, f func(*http.Request)) ([]byte, *http.Response, error) {
```

Comment:
instead of adding a new method, RawPost already exposes http.Response. At the call site you can

```
if err != nil {
  if resp != nil && resp.StatusCode == 404 {
     logger.Info(logtag, "diffing of cloud config is not supported by the server")
    //skip error
  } else {
    return err
  }
}
```